index.js

```diff
@@ -164,8 +164,7 @@ module.exports = (bookshelf, settings) => {
 
             return query
               .update(attrs, this.idAttribute)
-              .where(this.format(this.attributes))
-              .where(`${result(this, 'tableName')}.${settings.field}`, settings.nullValue)
+              .where(this.attributes)
```

Comment:
What query would you expect this to result in?

Reply:
Correct form is
```suggestion
              .where(this.idAttribute, this.get(this.idAttribute))
              .where(`${result(this, 'tableName')}.${settings.field}`, settings.nullValue)
```

which translates to `where id = (?) and where <tablename>.deleted_at = null`, which fits our purpose

`where` always works with two parts, left hand side and right hand side with an expression in between

where a = b
where a > b
where a is not null
where a <> b

etc